libnetdata/log/log.h

```diff
@@ -105,6 +105,19 @@ typedef struct error_with_limit {
     usec_t sleep_ut;
 } ERROR_LIMIT;
 
+typedef enum netdata_log_level {
+    NETDATA_LOG_LEVEL_INFO,
+    NETDATA_LOG_LEVEL_ERROR
+} netdata_log_level_t;
```

Comment:
The order should be opposite. E.g. (from lowest to highest)

 - ERROR
 - WARNING
 - INFO
 - NOTICE
 - DEBUG

Reply:
We only have two levels right now. I updated them